integration/commands_diagnostic_test.go

```diff
@@ -125,6 +127,60 @@ func TestCommandsDiagnosticGetLog(t *testing.T) {
 	}
 }
 
+func TestCommandsDiagnosticGetLogTelemetryReported(t *testing.T) {
```

Comment:
@AlekSi should this test actually be in `dance` as startup warnings for MongoDB look different? Or should we do smth like `skipForMongo` here?

Reply:
Let's cut that test for now